interfaces/chromium/chrome-common-extensions-api/autofill_private.idl

```diff
@@ -0,0 +1,356 @@
+// Copyright 2015 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Use the <code>chrome.autofillPrivate</code> API to add, remove, or update
+// autofill data from the settings UI.
+namespace autofillPrivate {
```

Comment:
Since namespace*Private are not really publicand are usable by the Extension developers, so adding to this repo would not be correct

Reply:
In response to this comment and [Rob's feedback](https://github.com/w3c/webextensions/pull/359#pullrequestreview-1329504658), I've stripped all `_private` and `_internal` files from this PR.